core/src/test/java/com/linecorp/armeria/common/SchemeTest.java

```diff
@@ -36,15 +37,25 @@ public void tryParse_null() {
     @Test
     void tryParse_add_none() {
         final Optional<Scheme> got = Scheme.tryParse("http");
-        assertThat(got.get().serializationFormat()).isEqualTo(SerializationFormat.NONE);
-        assertThat(got.get().sessionProtocol()).isEqualTo(SessionProtocol.HTTP);
+        final Condition<Scheme> condition = new Condition<>(scheme ->
```

Comment:
I think it's more idiomatic to call `assertThat` in the lambda instead of returning a boolean when using `satisfying` similar to here - https://joel-costigliola.github.io/assertj/core-8/api/org/assertj/core/api/AbstractOptionalAssert.html#hasValueSatisfying-java.util.function.Consumer-

Otherwise, all assertions get bundled into a single error message - `Condition` I think is more for framework authors than test writers.

```java
assertThat(Scheme.tryParse("http")).hasValueSatisfying(scheme -> {
  assertThat(scheme.sessionProtocol()).isEqualTo(HTTP);
  assertThat(scheme.serializationFormat()).isEqualTo(NONE);
});
```

Reply:
I was using _hasValueSatisfying(Condition<? super VALUE> condition)_.
But I see what the Javadoc says. Will update to use the Consumer function.

> Should be used as a way of deeper asserting on the containing object, as further requirement(s) for the value.

Reply:
Done